packages/components/src/ui/visually-hidden/hook.js

```diff
@@ -12,7 +12,7 @@ import * as styles from './styles';
 /** @typedef {import('../context').PolymorphicComponentProps<{}, 'div'>} Props */
 
 /**
- * @param {import('../context').PolymorphicComponentProps<{}, 'div'>} props
+ * @param {import('../context').PolymorphicComponentProps<{ children?: import('react').ReactNode }, 'div'>} props
```

Comment:
Should `children` be required on `VisuallyHidden`?

```suggestion
 * @param {import('../context').PolymorphicComponentProps<{ children: import('react').ReactNode }, 'div'>} props
```

Reply:
Done